cmd/kubectl/kubectlcobra/basic_printer.go

```diff
@@ -0,0 +1,47 @@
+package kubectlcobra
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/kustomize/kstatus/wait"
+)
+
+type BasicPrinter struct {
+	ioStreams genericclioptions.IOStreams
+}
+
+func (b *BasicPrinter) Print(ch <-chan Event) {
+	for event := range ch {
+		switch event.EventType {
+		case ErrorEventType:
+			cmdutil.CheckErr(event.ErrorEvent.Err)
+			return
+		case ApplyEventType:
+			obj := event.ApplyEvent.Object
+			gvk := obj.GetObjectKind().GroupVersionKind()
+			name := "<unknown>"
+			if acc, err := meta.Accessor(obj); err == nil {
+				if n := acc.GetName(); len(n) > 0 {
+					name = n
+				}
+			}
+			fmt.Fprintf(b.ioStreams.Out, "%s/%s %s\n", gvk.GroupKind().String(), name, event.ApplyEvent.Operation)
```

Comment:
can this go as a String() method on the event itself?  Or at least have a func String(Event) string